eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * File containing ReindexCommand class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
```

Comment:
you can skip this in your files, it is left over from long time ago that we don't use anymore.